mlflow/server/js/src/model-registry/components/ModelPage.enzyme.test.tsx

```diff
@@ -5,27 +5,25 @@
  * annotations are already looking good, please remove this comment.
  */
 
-import React from 'react';
 import configureStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
 import promiseMiddleware from 'redux-promise-middleware';
 import { mockModelVersionDetailed, mockRegisteredModelDetailed } from '../test-utils';
-import { ModelVersionStatus, Stages } from '../constants';
+import { ModelVersionStatus, Stages, MODEL_VERSIONS_PER_PAGE_COMPACT } from '../constants';
 import { Provider } from 'react-redux';
 import { MemoryRouter } from '../../common/utils/RoutingUtils';
 import { ModelPageImpl, ModelPage } from './ModelPage';
-import Utils from '../../common/utils/Utils';
 import { mountWithIntl } from '@mlflow/mlflow/src/common/utils/TestUtils.enzyme';
-import { ModelRegistryRoutes } from '../routes';
-import { ErrorWrapper } from '../../common/utils/ErrorWrapper';
 
 describe('ModelPage', () => {
-  let wrapper;
+  // @ts-expect-error TS(2709): Cannot use namespace 'ReactWrapper' as a type.
+  let wrapper: ReactWrapper;
```

Comment:
nit: is there an error if we let typescript infer the type here?
```suggestion
  let wrapper;
```